src/interface/initialize.py

```diff
@@ -0,0 +1,12 @@
+import streamlit as st
+
+
+def initialize_variables() -> None:

```

Comment:
I believe this should be some class method, but actually we can keep this one in init, not create separate function.